src/components/ChatList/ChannelListHeaderItem.tsx

```diff
@@ -54,7 +54,7 @@ const ChannelListHeaderItem = (props: ChannelListTabItemProps) => {
         <FastImage source={picture} style={[styles.picture, styles.anonBgColor]} />
       )}
       <Text numberOfLines={1} ellipsizeMode="tail" style={styles.name}>
-        as {name}
+        {type === ANONYMOUS ? name : `as ${name}`}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The text display logic has been updated to show different text based on the `type` value. This change is good as it provides a more dynamic way of displaying user names based on their type (anonymous or signed).
```diff
-        as {name}
+        {type === ANONYMOUS ? name : `as ${name}`}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->